charts/eck-resources/examples/production.yaml

```
@@ -0,0 +1,107 @@
+fullnameOverride: eck
+
+elasticsearch:
+  enabled: true
+  credentials:
+    enabled: true
+  version: 7.16.3
+  secretConfig:
+    xpack.security.authc.realms.oidc.dex.rp.client_secret: test
+  config:
+    node.store.allow_mmap: false
+    ingest.geoip.downloader.enabled: false
+    xpack.security.http.ssl.enabled: true
+  nodeSets:
+  - name: data
+    count: 3
+    config:
+      node:
+        roles: [data, ingest]
+    resources:
+      requests:
+        cpu: "2"
+        ephemeral-storage: 100Mi
+        memory: "4Gi"
+      limits:
+        cpu: "4"
+        ephemeral-storage: 100Mi
+        memory: "4Gi"
+    env:
+    - name: ES_JAVA_OPTS
+      value: "-Xmx2g -Xms2g -XX:MaxRAM=3072m"
+    volumeClaimTemplates:
+    - metadata:
+        name: elasticsearch-data # Do not change this name unless you set up a volume mount for the data path.
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 200Gi
+        storageClassName: "staging-performance"
```

If this is an example for a production environment, this should probably be:

```suggestion
        storageClassName: "production-performance"
```